site/static/js/main.js

```diff
@@ -18,7 +18,7 @@ function initializeWishlist() {
       originalFunction.apply(this, [className]);
 
       /* Convert this to a swiper */
-      let containers = wishlist.getElementsByClassName('swiper-container');
+      let containers = wishlist.getElementsByClassName('collection-container');
```

Comment:
Is it a good idea to be breaking changing all of these swiper classes?  This will be an upgrade blocker for the NZ Rugby site... not sure about others.

Reply:
You could keep the swiper classes on the swipers for use in JS but use your new classes for CSS.... that would be slightly less breaky.

Reply:
That was a good idea actually. Changed.

We defo need the sliders in relish. I'll bring it up.